### PR TITLE
GIX-1860: Change Wallet Header

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Adapt the wallet pages to the new header UI.

# Changes

* Use the new WalletPageHeader and WalletPageHeading in NnsWallet.
* Use the new WalletPageHeader and WalletPageHeading in SnsWallet.
* Use the new WalletPageHeader and WalletPageHeading in CkBTCWallet.
* Add Tooltip with detailed balance to WalletPageHeading.
* Remove the menu bar from CkBTCWalletActions. Not needed anymore.
* Remove padding top in IcrcTransactionsList. Not needed anymore.
* Wrap TransactionsList with an element to prevent the parent from setting the spacing of the elements.

# Tests

* Add a test in NnsWallet page to check that HW buttons are displayed.
* Remove tests regarding menu bar in CkBTCWalletActions.
* Fix NnsWallet.spec, SnsWallet.spec and CkBTCWallet.spec

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

SNS Wallet
Screenshot 2023-10-09 at 08 52 08

NNS Wallet: Main
Screenshot 2023-10-09 at 08 52 16

NNS Wallet: Subaccount
Screenshot 2023-10-09 at 08 52 22

NNS Wallet: HW account
Screenshot 2023-10-09 at 08 52 30

CkBTC Wallet
Screenshot 2023-10-09 at 08 52 39

**Comment by lmuntaner** [comment]
@peterpeterparker please review

File: frontend/src/lib/pages/CkBTCWallet.svelte [hunk 1]
```diff
@@ -228,3 +238,11 @@
     <CkBTCWalletFooter />
   {/if}
 </Island>
+
+<style lang="scss">
```
Comment (peterpeterparker) [inline frontend/src/lib/pages/CkBTCWallet.svelte:242]: not for this PR but this section style sounds like something we should apply globally

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: frontend/src/lib/pages/CkBTCWallet.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/pages/CkBTCWallet.svelte:242]: Yes, I was thinking. Not to all sections, something like a SectionDetailPage.

**Comment by lmuntaner** [comment]
@peterpeterparker tests weren't passing. I committed again fixing them if you want to take another look.

File: frontend/src/lib/components/accounts/WalletPageHeading.svelte [hunk 2]
```diff
@@ -35,15 +45,24 @@
 </script>
 
 <PageHeading testId="wallet-page-heading-component">
-  <TestIdWrapper slot="title" testId="wallet-page-heading-title">
-    {#if nonNullish(balance)}
-      <AmountDisplay amount={balance} size="huge" singleLine />
+  <svelte:fragment slot="title">
+    <!-- TS is not smart enough to understand that if `balance` is defined, then `detailedAccountBalance` is also defined -->
```
Comment (peterpeterparker) [inline frontend/src/lib/components/accounts/WalletPageHeading.svelte:49]: You can use a `@const` instead?

```
{#if nonNullish(balance)
  {@const detailedAccountBalance = formatToken...
```

**Review by peterpeterparker — APPROVED** [review]
lgtm, thanks

File: frontend/src/lib/components/accounts/WalletPageHeading.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/lib/components/accounts/WalletPageHeading.svelte:49]: I would need to cast it. I don't think that's a better way.

Ideally, I would create an object where both are defined, or non is. I don't think it's worth this extra complexity.

Comment (peterpeterparker) [reply frontend/src/lib/components/accounts/WalletPageHeading.svelte:48]: Sounds good

**Pull request merged**